beta-20160825.md

```diff
@@ -0,0 +1,64 @@
+---
+title: What's New in beta-20160825
+toc: false
+summary: Additions and changes in CockroachDB version beta-20160825.
+---
+
+## Aug 25, 2016
+
+### General Changes
+
+- CockroachDB now uses Go 1.7. [#8579](https://github.com/cockroachdb/cockroach/pull/8579)
+
+### Command-Line Interface Changes
+
+- The `cockroach zone set` no longer takes its input directly on the command line. Instead, it accepts a `--file` flag to read from a file, or `--file=-` to read from standard input. [#7953](https://github.com/cockroachdb/cockroach/pull/7953)
+
+### New Features
+
+- `AS OF SYSTEM TIME` can now be used with `JOIN` queries. [#8198](https://github.com/cockroachdb/cockroach/pull/8198)
+- The type `BIT` now works correctly (as a shorthand for `BIT(1)`). The length limit is now enforced. [#8326](https://github.com/cockroachdb/cockroach/pull/8326)
+- The `SHOW` commands now only show tables that the current user has permission to access. [#8070](https://github.com/cockroachdb/cockroach/pull/8070)
+- Foreign keys can now use a prefix of an index. [#8059](https://github.com/cockroachdb/cockroach/pull/8059)
+- The standard `information_schema` database is now partially supported. New tables will be added to this database in future beta releases. [#7965](https://github.com/cockroachdb/cockroach/pull/7965), [#8119](https://github.com/cockroachdb/cockroach/pull/8119)
+
+### Bug Fixes
+
+- The `RENAME` command now requires the `DROP` privilege on the table or database. It is no longer possible to rename the `system` database. [#7998](https://github.com/cockroachdb/cockroach/pull/7998)
+- The `repeat()` and `substr()` functions with very large numeric arguments will no longer crash the server as easily. [#8073](https://github.com/cockroachdb/cockroach/pull/8073), [#8078](https://github.com/cockroachdb/cockroach/pull/8078)
+- Certain errors now cause a replica to be marked as corrupt so the corruption will not be replicated to other nodes. [#7684](https://github.com/cockroachdb/cockroach/pull/7684)
+- A `CloseComplete` packet is now sent in response to a `Close` command, improving compatibility with come PostgreSQL client drivers (including the Elixir driver). [#8309](https://github.com/cockroachdb/cockroach/pull/8309)
+- SQL name parsing has been improved to fix several panics and allow foreign key constraints that span databases. [#8152](https://github.com/cockroachdb/cockroach/pull/8152)
+- Fixed some panics. [#8283](https://github.com/cockroachdb/cockroach/pull/8283), [#8282](https://github.com/cockroachdb/cockroach/pull/8282)
+- Fixed a deadlock if a range grew too large. [#8387](https://github.com/cockroachdb/cockroach/pull/8387)
```

Comment:
Yes. The commit message is "fix data race and deadlock"; the PR message was not updated.